scylla-gdb.py

```diff
@@ -5902,6 +5903,7 @@ def invoke(self, args, from_tty):
             semaphores = [db["_read_concurrency_sem"], db["_streaming_concurrency_sem"], db["_system_read_concurrency_sem"]]
             try:
                 semaphores.append(db["_compaction_concurrency_sem"])
+                semaphores.append(db["_view_update_read_concurrency_sem"])
```

Comment:
Put this in a separate try-catch and add a comment: # < 6.2
So we know when we can remove the respective try-catch.

Reply:
Added a separate try-catch clause in the rebase